### DISCUSSION TITLE
А как запустить?

### DISCUSSION THREAD

**Discussion by MiddleFromDiscord** [bol-van/zapret#778] [Q&A]
Я, конечно, понимаю, что такой сделали эту программу, но я не понимаю как её собственно говоря открыть?

**Comment by Alexsuper8783** [comment]
https://github.com/bol-van/zapret/blob/master/docs/quick_start.md для linux и openwrt
https://github.com/bol-van/zapret/blob/master/docs/quick_start_windows.md для windows

**Reply by MiddleFromDiscord** [reply]
Спасибо, попробую.

**Comment by trollen28** [comment]
23.11.2024 у меня кончился ютуб.
Я картошка, но пробовал по инструкции https://github.com/bol-van/zapret/blob/master/docs/quick_start_windows.md
мои действия :
вырубил расширения в браузере
запустил zapret-winws/preset_russia.cmd ютуба не пашет
запустил windivert_delete.cmd
запустил blockcheck\blockcheck.cmd дальше по инструкции туман и прошлось идти за пивом.
запустил zapret-winws/preset_russia.cmd заработало.
PROFIT
Смотрю свои уроки по аккордеону на ютубе и кланяюсь автору шайтан-машины.
